src/api/donations.py

```
@@ -0,0 +1,618 @@
+                if isinstance(item, dict):
+                    pairs.extend(_stripe_encode(item, item_key))
+                elif item is not None:
+                    pairs.append((item_key, str(item)))
+        elif isinstance(value, bool):
+            pairs.append((full_key, "true" if value else "false"))
+        elif value is not None:
+            pairs.append((full_key, str(value)))
+    return pairs
+
+
+async def _stripe(method: str, path: str, data: dict, key: str):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_stripe`

(ANN202)